monitoring-alerting/helm/values.yml

```diff
@@ -0,0 +1,50 @@
+grafana:
+  persistence:
+    enabled: true
+
+  grafana.ini:
```

Comment:
I wonder if we can use the `--set-file` help argument (e.g. `helm install ... --set-file grafana.grafana.ini=<some local file>`) to handle the contents of the section below so that we don't need to persist a FQDN or client_id/client_secret but still maintain the use of a `values.yaml` file?

Reply:
If we went with the local file route wouldn't we want to include that in the repo as well?

I agree that I don't like the FQDN being persisted.

Reply:
Similar to [this comment](https://github.com/homedepot/spingo/pull/62#discussion_r361997575) it seems as if we could handle sensitive fields like client_id/client_secret/domain name) ruing the helm installation/upgrade via a script or pipeline instead of persisting the values to the repo.  What do you think?

Reply:
agreed, notice that we are creating a dynamic vault valuesyaml file for helm here https://github.com/homedepot/spingo/blob/master/spinnaker/modules/vault/vault.tf#L38-L55 then populating it when we create the halyard vm

Reply:
I created a simple `install.sh` script to handle these fields so they're no longer persisted to the repo.